app/main.py

```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        {
+            "code": "name",
+            "text": "message",
+            "line_number": "line",
+            "column_number": "column",
+            "filename": "source"
+        }[signature]: ("flake8" if signature == "filename" else sense)
+        for signature, sense in error.items()
+        if signature != "physical_line"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [format_linter_error(error) for error in errors],
+            "path": file_path,
+            "status": "failed" if errors else "passed"}
```

Comment:
While creating a dictionary — write key-value pairs in a single row. The curly braces must be located in one of two options: open and start with the text or have a line break between the text.